.idea/compiler.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
I think you should probably gitignore the .idea files since they are related to your own intelliJ setup. If the gitignore doesnt work (had some problems with that before), try using the built in git tool in intelliJ, it automatically ignores all of these when committing.

Reply:
.DS_Store as well